l10n_it_withholding_tax/views/report_invoice.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="print_withholding_tax" model="ir.ui.view">
+        <field name="name">print_withholding_tax</field>
+        <field name="inherit_id" ref="account.report_invoice_document"/>
+        <field name="arch" type="xml">
+            <xpath expr="//table[@name='invoice_line_table']/thead/tr/th[last()]" position="before">
```

Comment:
Forse meglio non usare `last()` ma magari identificare l'elemento per numero, perchè potrebbe sbagliare:
![image](https://user-images.githubusercontent.com/1033131/114893160-bbf39b80-9e0d-11eb-88f5-821a007e0480.png)

Reply:
Corretto.

Reply:
Mi sembra ci siano ancora problemi:

![image](https://user-images.githubusercontent.com/1033131/115238790-61648300-a11e-11eb-82ee-1358f3ed5c1d.png)

Hai provato?

Reply:
Avevo provato in locale prima di fare il push e generava una tabella corretta. Adesso alla prima opportunità ci riguardo...

Reply:
Ho ricontrollato e a me in locale continuava a vedersi correttamente. In ogni caso, adesso ho specificato il numero del td/th in entrambi gli xpath così, spero, non ci saranno problemi di disallineamento tra header della tabella e corpo.

Reply:
E invece continua a dare lo stesso problema su Runbot, anche se in locale funziona...

Reply:
Ho trovato il problema (e perchè in locale per me non si presentava), ed è legato al modulo account_invoice_report_ddt_group, e a questo file in particolare:

https://github.com/OCA/l10n-italy/blob/ddb923f124930a740e7040b7e941f2055b3d934c/account_invoice_report_ddt_group/views/invoice_ddt.xml#L3-L19

Tra l'altro, la visualizzazione è comunque corretta se si installa prima withholding tax piuttosto che report_ddt_group.

Il problema si risolve se invece delle righe 7-9 di invoice_ddt si scrive

```
<t t-if="has_serial_number">
    <th name="lot_serial">
        <strong>Lots/Serial Numbers</strong>
    </th>
 </t>
```

Devo includere questa fix in questa stessa PR o aprirne una nuova?

Reply:
@aleuffre grazie mille per l'analisi.
Dovresti aprire un'altra PR per il modulo `account_invoice_report_ddt_group` , con riferimento a questa